src/services/WolfDen.Domain/Entity/AttendenceClose.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class AttendenceClose

```

Comment:
Purpose of this table

Reply:
HR can close attendance for a month, all leaves after that will be LOP days. So inorder to mark the attendance close date